core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -164,6 +166,11 @@ private void registerPublicKeysRoute(
         .failureHandler(errorHandler);
   }
 
+  private void registerStaticRoute(final Router router) {
+    router.route("/openapi/*").handler(StaticHandler.create("openapi"));
+    LOG.info("Registered static route /openapi");
```

Comment:
nit: not sure is much value in this log statement

Reply:
agreed!